sympy/ntheory/tests/test_generate.py

```diff
@@ -142,6 +148,21 @@ def test_generate():
     assert prevprime(97) == 89
     assert prevprime(10**40) == (10**40 - 17)
 
+    for sieve_interval in [1, 10, 11, 1_000_000]:
+        s = Sieve(sieve_interval=sieve_interval)
+        assert s._list[-1] == 13
```

Comment:
Do I understand correctly that `sieve_interval` is the size by which we let the sieve grow (but it always starts with 6 elements, the last being 13)?

Reply:
That's right. The `sieve_interval` is the length of the array to allocate in one sieve. Normally, it should be set to some large value, but setting it to `1` works correctly in theory.

The `s._list[-1] == 13` check was intended to make the precondition explicit. If the default value of `_list` were changed to `[2, 3, 5, 7, 11]`, the program would still work correctly, but would no longer satisfy the test condition. This is misleading and has been fixed.